packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -20,57 +16,75 @@
   border-radius: var(--border-radius-medium);
   font-family: var(--font-stack-primary);
   font-size: calc(1rem * (14 / var(--font-base-default)));
+  font-weight: 500;
   overflow: hidden;
   background-size: 100% 200%;
-  transition: background var(--transition-duration-default)
+  transition: background var(--transition-duration-short)
       var(--transition-easing-default),
     opacity var(--transition-duration-default) var(--transition-easing-default),
     border-color var(--transition-duration-default)
       var(--transition-easing-default);
   vertical-align: middle;
   text-decoration: none;
+  box-shadow: 0px 1px 0px rgb(25, 37, 50, 0.1);
```

Comment:
I think it's better to put these values in tokens? maybe we will use them later for focusable notifications? `box-shadow-positive`, ..etc? wdyt

Reply:
Yes, I can create a token for it!